docs/tutorial/tutorial.md

```diff
@@ -0,0 +1,129 @@
+# Tutorial
+
+### Prerequisites
+
+To deploy wordpress-k8s charm, you will need a juju bootstrapped with any kubernetes controller.
+To see how to bootstrap your juju installation with microk8s, please refer to the documentation
+on microk8s [installation](https://juju.is/docs/olm/microk8s).
+
+### Setting up the environment
+
+To easily clean up the resources and to separate your workload from the contents of this tutorial,
+it is recommended to set up another environment by adding a new model with the following command.
+
+```
+juju add-model wordpress-tutorial
+```
+
+### Deploy wordpress-k8s charm
+
+Deployment of WordPress requires a database integration. A preferred method in juju is through the
+native juju relation. In this case, a `mysql` interface is required by the wordpress-k8s charm and
+hence, [`mysql-k8s`](https://charmhub.io/mysql-k8s) charm will be used.
+
+Start off by deploying the wordpress charm. By default it will deploy the latest stable release of
+the wordpress-k8s charm.
+
+```
+juju deploy wordpress-k8s
+```
+
+If you wish to try out the latest features of the wordpress-k8s charm you can set an additional
```

Comment:
Would it be worth omitting this to reduce the complexity of the tutorial?

Reply:
Makes sense, simpler the better :)